src/components/ErrorSuppression/ErrorSuppressionFilters/ErrorSuppressionFilters.js

```diff
@@ -205,23 +205,33 @@ export const ErrorSuppressionFilters = () => {
   };
 
   const onFacilityChange = (value) => {
-    setSelectedFacility(value);
+    setSelectedFacility(value === null ? null : value);
```

Comment:
This is unnecessary, it always evaluates to `value`.
```suggestion
    setSelectedFacility(value);
```